Code/GraphMol/molopstest.cpp

```diff
@@ -7655,6 +7655,302 @@ void testGithub1990() {
   }
   BOOST_LOG(rdInfoLog) << "Finished" << std::endl;
 }
+
+void testRemoveAndTrackIsotopes() {
```

Comment:
again, just a comment, not a request for anything to be changed.
This test is almost as complex as the code it's testing. I think it's a good idea to keep the testing code itself as simple as possible (it may be that this is true here) so that we don't end up in the awkward situation of having to debug our tests.

Reply:
In general, I agree with you. In this specific case, I felt that adding a more complex test case with comments would remind me of what I wanted to achieve with the code better than a test case that just checks the outcomes. I promise not to do this too often : )